compose/install.md

```diff
@@ -109,16 +109,16 @@ target="_blank" class="_"}. Follow the instructions from the link, which involve
 running the `curl` command in your terminal to download the binaries. These step
 by step instructions are also included below.
 
-1.  Run this command to download the latest version of Docker Compose:
+1. a. Run this command to download the latest Stable version of Docker Compose **(Recommended for production)**:
 
     ```bash
-    sudo curl -L "https://github.com/docker/compose/releases/download/{{site.compose_version}}/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+    sudo curl -L `curl -s 'https://api.github.com/repos/docker/compose/releases' | grep -e 'browser_download_url' | grep "docker-compose-$(uname -s)-$(uname -m)\"\$" | head -n1 |cut -d'"' -f4` -o /usr/local/bin/docker-compose
```

Comment:
This change looks similar to https://github.com/docker/docker.github.io/pull/7061, and I'm not sure we should add this complexity (see my comment on https://github.com/docker/docker.github.io/pull/7061#issuecomment-453153904)

If the currently set version of `site.compose_version` is incorrect, we should fix that instead, and make sure that whenever a new release is done, a corresponding pull-request is opened for the docs.

/cc @chris-crone @ijc @ulyssessouza ptal

Reply:
Thanks for your contribution @Maannniii!

I agree with @thaJeztah though, we should rely on `site.compose_version`.

Reply:
but @thaJeztah @chris-crone `site.compose_version` doesn't reflect pre-releases. I believe the word **Latest ** misguides the user. Here we should mention something like **Latest stable** or pre-releases are not included.

Reply:
I opened a PR with an alternative approach for this; https://github.com/docker/docker.github.io/pull/8411